assets/metainfo/tuned-switcher.metainfo.xml.in

```diff
@@ -12,7 +12,10 @@ SPDX-License-Identifier: CC0-1.0
     <description>
         <p>
             Tuned Switcher is a simple utility to manipulate the Tuned service.
+        </p>
+        <p>
             Tuned is daemon for monitoring and adaptive tuning of system devices.
+            The daemon must be installed on your system for this program to function.
```

Comment:
Please change "for this program to function" to "in order to use this application".

Reply:
Ok now?